Index: src/org/hbrs/se1/ws24/exercises/uebung4/Client.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/org/hbrs/se1/ws24/exercises/uebung4/Client.java b/src/org/hbrs/se1/ws24/exercises/uebung4/Client.java
new file mode 100644
--- /dev/null	(date 1730210739582)
+++ b/src/org/hbrs/se1/ws24/exercises/uebung4/Client.java	(date 1730210739582)
@@ -0,0 +1,4 @@
+package org.hbrs.se1.ws24.exercises.uebung4;
+
+public class Client {
+}
